doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -0,0 +1,103 @@
+[id='operator-deploy-kieserver-proc']
+= Setting custom {KIE_SERVER} configuration for the environment
```

Comment:
"a" is not happening here because actually one can create multiple KIE servers here in a "rinse and repeat" step. But it turns out I forgot the step, which was not caught in SME review. I am adding it now.